src/io/eval.rs

```diff
@@ -238,13 +238,17 @@ where
                 encode.plaintext.clone()
             };
             log_mem(format!("plaintext at {}, {} computed", level, j));
+            // C_att, xL: new encoding of next level
             let new_encode: BggEncoding<M> =
                 BggEncoding::new(new_vec, pub_key_level[j].clone(), plaintext);
             log_mem(format!("new_encode at {}, {} computed", level, j));
             new_encodings.push(new_encode);
         }
+        // s_xL: input dependent secret key
         ps.push(p.clone());
```

Comment:
vector `p` is a function of `s` but not equal as the input dependent secret key `s`